test/e2e/framework/util.go

```diff
@@ -501,7 +501,9 @@ func LoadClientset() (*clientset.Clientset, error) {
 
 // RandomSuffix provides a random sequence to append to pods,services,rcs.
 func RandomSuffix() string {
-	return strconv.Itoa(rand.Intn(10000))
+	nonce := time.Now().Nanosecond()
+	// Reduce the chance of collision
+	return strconv.Itoa(nonce) + strconv.Itoa(rand.Intn(10000))
```

Comment:
wondering if this might be too long? but it should be okay I guess...

Reply:
According to https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names, the max length of the name is 253, I think it is okay.

Reply:
hmmm it is actually too long in other scenarios.. https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/94247/pull-kubernetes-e2e-kind-ipv6/1298521049540857857

Reply:
reverted